{{cookiecutter.github_repository}}/docs/graphql/errors_handling.md

```diff
@@ -0,0 +1,126 @@
+# Error Handling
+
+There are several error types in the GraphQL API, and you may come across different ones depending on the operations you are trying to perform.
+
+The GraphQL API handles the following three types of errors:
+
+## Query-level errors
+
+This error occurs if you provide wrong or unrecognized input data while performing a specified operation. GraphQL checks the syntax as you write, and if you are trying to execute an unknown operation, the editor you are using will notify you. If you proceed with sending the request, you will get a syntax error.
+
+Below is an example of an error triggered by the wrong syntax. The following query tries to fetch the fullName field, which doesn't exist on the User type:
+
+
+```
+query {
+  me {
+    fullName
+  }
+}
+```
+
+Sending this query to the server would result in the following syntax error:
+
+```json
+{
+  "error": {
+    "errors": [
+      {
+        "message": "Cannot query field \"fullName\" on type \"User\". Did you mean \"firstName\" or \"lastName\"?",
+        "locations": [
+          {
+            "line": 3,
+            "column": 5
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Data-level errors
+
+This error occurs when the user passes invalid data as the mutation input. For example, you provide an email address already used in another user's account while creating a new user. It is therefore not unique, and as a result, you will get a validation error.
```

Comment:
```suggestion
This error occurs when the user passes invalid data as the mutation input. For example, using an email address that is associated with a user account to create a secondary account will throw a validation error since the email should be unique within a user account.
```